hw/ip/entropy_src/data/entropy_src_sec_cm_testplan.hjson

```diff
@@ -24,70 +24,127 @@
 {
   testpoints: [
     {
+      // TODO: We need to carefully check what points are covered by which test and how.
+      // At a first glance 1), 2) and 3b) should be covered by the automated entropy_src_csr_rw test.
+      // However, the SW_REGUPD and MODULE_ENABLE registers have exclusions for all automated CSR tests.
+      // Meaning we might need a directed test for 1), 2) and 3b).
+      // 3a) seems to be captured by the scoreboard.
```

Comment:
Note that we have issue #16065 to track the remaining work at a higher level but I suggest to keep these TODOs for now and remove them as we extend the tests throughout the next couple of days.